# HG changeset patch
# User sgehwolf
# Date 1523874036 -7200
#      Mon Apr 16 12:20:36 2018 +0200
# Node ID fc3a72e286e0d5bf9b994d1b778b39f3b080c686
# Parent  69bae761600bdbc97953500ac51a396d31f54ee9
8201495: [Zero] Reduce limits of max heap size for boot JDK on s390
Reviewed-by: ihse, aph, simonis

diff --git a/make/autoconf/boot-jdk.m4 b/make/autoconf/boot-jdk.m4
--- a/make/autoconf/boot-jdk.m4
+++ b/make/autoconf/boot-jdk.m4
@@ -405,7 +405,7 @@
   BOOTCYCLE_JVM_ARGS_BIG=-Xms64M
 
   # Maximum amount of heap memory and stack size.
-  JVM_HEAP_LIMIT_32="1024"
+  JVM_HEAP_LIMIT_32="768"
   # Running a 64 bit JVM allows for and requires a bigger heap
   JVM_HEAP_LIMIT_64="1600"
   STACK_SIZE_32=768
